test-files/cmd/slow/ConflateCmdStatsAllDataTypesTest.sh.stdout

```diff
@@ -15,10 +15,10 @@ Features Conflatable by: hoot::ScriptMatchCreator,PoiGeneric.js	11	10
 Total Conflated Features	0	0	18	
 Percentage of Total Features Conflated	0	0	45	
 Total Features Marked for Review	0	0	11	
-Percentage of Total Features Marked for Review	0	0	27.500000000000004	
+Percentage of Total Features Marked for Review	0	0	27.5	
 Total Number of Reviews to be Made	0	0	8	
 Total Unmatched Features	25	22	22	
-Percentage of Total Features Unmatched	100	100	55.000000000000007	
+Percentage of Total Features Unmatched	100	100	55.00000000000001	
```

Comment:
Interesting how you are getting one less significant digit on all of your floats from here on out.  I assume that translates to the build machines as well as any other build box or we could have some issues.